packages/swapper/src/swappers/ChainflipSwapper/swapperApi/getTradeQuote.ts

```diff
@@ -330,7 +330,8 @@ export const _getTradeQuote = async (
           sellAsset,
           accountNumber,
           allowanceContract: '0x0', // Chainflip does not use contracts - all Txs are sends
-          estimatedExecutionTimeMs: singleQuoteResponse.estimatedDurationSeconds! * 1000,
+          estimatedExecutionTimeMs: 
+            (singleQuoteResponse.estimatedDurationsSeconds!.deposit! + singleQuoteResponse.estimatedDurationsSeconds!.swap!) * 1000,
```

Comment:
I didn't stop on those initially but, why are we assuming those are defined? The generated types tells us it can be undefined, wouldn't it be safer to early return if these are not?

Reply:
Some weird thing going on with the generated types, but these always have values, I doubled checked with a core chainflip dev:

![image](https://github.com/user-attachments/assets/0192a70d-018c-4ef4-bc6f-45813f886860)

Reply:
Ok, are those types intended to change at any time? cause we could simply remove the optional chaining from them and so from our codebase

Reply:
Nope! We can do that indeed

Reply:
Unrelated to this PR so please don't push it there, but if at any time we have to refactor it or change something related we will fix it